### DISCUSSION TITLE
Move "View Origin and Grid" to "Perspective" tab for consistency

### DISCUSSION THREAD

**Discussion by catboy-catfish** [godotengine/godot-proposals#9426] [Editor]
Would be cool if you guys could move the "View Origin" and "View Grid" from the View tab to the Perspective menu in the upper left corner of the scene window, preferably amongst the options "View Environment, Gizmos, Frame Time and Information". This way I think it's more consistent.

**Comment by timothyqiu** [comment]
These options affect the whole scene. But perspective menu items are per viewport.

**Reply by catboy-catfish** [reply]
Thanks for the info.

**Reply by catboy-catfish** [reply]
Wait, nvm, can't we just change these settings from per-scene to per-viewport?

**Reply by Calinou** [reply]
In 4.3.dev, **View Grid** is available in the Perspective menu already. It lets you toggle the grid on a per-viewport basis, while the **View** menu affects all viewports at once.

**Reply by catboy-catfish** [reply]
@Calinou Thank you for letting me know. Maybe the reason I didn't know that earlier was because I'm rather behind (I use Godot 4.2.2 RC custom). I do wanna upgrade to 4.3 but Godot Jolt hasn't come out for 4.3 yet
